cmd/loadTests.go

```diff
@@ -114,6 +212,12 @@ func setup(cmd *cobra.Command, args []string) {
 	cmd.SilenceUsage = true
 	term := terminal.New(cmd.InOrStdin, cmd.OutOrStdout, verbose)
 
+	/*
+		used for the json output file -
+		loadTestsTimestamp - loadTests start timestamp, used for the json output file
+	*/
+	loadTestsTimestamp := time.Now().Format("2006/01/02 15:04:05")
```

Comment:
Can we use format like `2006-01-02T15:04:05Z07:00`?

Reply:
Sure